ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/DefaultSynchronizer.java

```diff
@@ -126,7 +126,16 @@ public void start() {
       LOG.info("Starting synchronizer.");
       blockPropagationManager.start();
       if (fastSyncDownloader.isPresent()) {
-        fastSyncDownloader.get().start().whenComplete(this::handleFastSyncResult);
+        fastSyncDownloader
+            .get()
+            .start()
+            .whenComplete(this::handleFastSyncResult)
+            .exceptionally(
+                ex -> {
+                  System.exit(0);
+                  return null;
```

Comment:
We have a `stop` method here, we should probably be calling that instead of just exiting. There could be important resources to close or cleanup nicely now or in the future.

Reply:
I think when we call `System.exit` the [shutdown hook](https://github.com/mbaxter/besu/blob/26af974913266363eea994af52ffdb21a603881e/besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java#L1948-L1961) will get invoked (which stops everything cleanly), but worth double-checking.

Reply:
![image](https://user-images.githubusercontent.com/6472741/83667816-49037c80-a5cf-11ea-9539-1ae348fb1cda.png)
As you may see, the `stop` method is correctly triggered when the `System.exit` is called.